extras/kdf-keys/pbkdf2.c

```diff
@@ -105,9 +108,18 @@ int main(int argc, char *argv[])
 		exit(1);
 	}
 
-	fscanf(stdin, "%ms", &pass);
-	if ( pass[strlen(pass)-1] == '\n' )
-		pass[strlen(pass)-1] = '\0';
+	int j = 0;
```

Comment:
Wouldn't it made more sense to name this var something like pw_lenght, or else?
Passing `j-1` to gcry_kdf_derive isn't descriptive, without further looking?

Reply:
👍 It makes sense for sure